src/Api/SecretsManager/Models/Response/AccessPolicyResponseModel.cs

```diff
@@ -9,161 +9,133 @@ public abstract class BaseAccessPolicyResponseModel : ResponseModel
         : base(accessPolicy, _objectName)
     {
         GroupId = accessPolicy.GroupId;
-        GrantedProjectId = accessPolicy.GrantedProjectId;
         GroupName = accessPolicy.Group?.Name;
         CurrentUserInGroup = accessPolicy.CurrentUserInGroup;
     }
 
-    public GroupProjectAccessPolicyResponseModel() : base(new GroupProjectAccessPolicy(), _objectName)
+    public GroupAccessPolicyResponseModel() : base(new GroupProjectAccessPolicy(), _objectName)
     {
     }
 
     public Guid? GroupId { get; set; }
     public string? GroupName { get; set; }
     public bool? CurrentUserInGroup { get; set; }
-    public Guid? GrantedProjectId { get; set; }
 }
 
-public class GroupServiceAccountAccessPolicyResponseModel : BaseAccessPolicyResponseModel
+public class ServiceAccountAccessPolicyResponseModel : BaseAccessPolicyResponseModel
 {
-    private const string _objectName = "groupServiceAccountAccessPolicy";
+    private const string _objectName = "serviceAccountProjectAccessPolicy";
 
-    public GroupServiceAccountAccessPolicyResponseModel(GroupServiceAccountAccessPolicy accessPolicy)
+    public ServiceAccountAccessPolicyResponseModel(ServiceAccountProjectAccessPolicy accessPolicy)
         : base(accessPolicy, _objectName)
     {
-        GroupId = accessPolicy.GroupId;
-        GroupName = accessPolicy.Group?.Name;
-        GrantedServiceAccountId = accessPolicy.GrantedServiceAccountId;
-        CurrentUserInGroup = accessPolicy.CurrentUserInGroup;
+        ServiceAccountId = accessPolicy.ServiceAccountId;
+        ServiceAccountName = accessPolicy.ServiceAccount?.Name;
     }
 
-    public GroupServiceAccountAccessPolicyResponseModel() : base(new GroupServiceAccountAccessPolicy(), _objectName)
+    public ServiceAccountAccessPolicyResponseModel(ServiceAccountSecretAccessPolicy accessPolicy)
+        : base(accessPolicy, _objectName)
     {
+        ServiceAccountId = accessPolicy.ServiceAccountId;
+        ServiceAccountName = accessPolicy.ServiceAccount?.Name;
     }
 
-    public Guid? GroupId { get; set; }
-    public string? GroupName { get; set; }
-    public Guid? GrantedServiceAccountId { get; set; }
-    public bool? CurrentUserInGroup { get; set; }
+    public ServiceAccountAccessPolicyResponseModel()
+        : base(new ServiceAccountProjectAccessPolicy(), _objectName)
+    {
+    }
+
+    public Guid? ServiceAccountId { get; set; }
+    public string? ServiceAccountName { get; set; }
 }
 
-public class ServiceAccountProjectAccessPolicyResponseModel : BaseAccessPolicyResponseModel
+public class GrantedProjectAccessPolicyResponseModel : BaseAccessPolicyResponseModel
 {
     private const string _objectName = "serviceAccountProjectAccessPolicy";
 
-    public ServiceAccountProjectAccessPolicyResponseModel(ServiceAccountProjectAccessPolicy accessPolicy)
+    public GrantedProjectAccessPolicyResponseModel(ServiceAccountProjectAccessPolicy accessPolicy)
```

Comment:
update `_objectName`

Reply:
Great catch, updated in https://github.com/bitwarden/server/pull/4146/commits/cc7f5dd85e2e29873417c53e5c134685c0f27aa3